builder/Dockerfile.debian-10

```diff
@@ -17,6 +17,9 @@ RUN chmod 0777 /opt
 # Pin fpm to avoid git dependency in 1.12.0
 RUN gem install fpm:1.11.0
 
+# Make sure that patching the OS does not break Java
+ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64/
```

Comment:
On Ubuntu/Debian, the R builds here appear to already use this major-versioned location:
```sh
R CMD config JAVA_HOME
# /usr/lib/jvm/java-11-openjdk-amd64
```

Checking the default Ubuntu/Debian R packages, they appear to all use `/usr/lib/jvm/default-java` from `default-jdk` or `default-jre`:
```sh
R CMD config JAVA_HOME
# /usr/lib/jvm/default-java
```
Unfortunately it doesn't look like you can control the JRE location using the alternatives system in Ubuntu/Debian. `/usr/lib/jvm/default-java` is put there by `default-jdk`/`default-jre`, and not present if you install Java 11 or Java 8 specifically.
```sh
ls /usr/lib/jvm -la 
# lrwxrwxrwx 1 root root   25 Jul 17  2019 default-java -> java-1.11.0-openjdk-amd64
# lrwxrwxrwx 1 root root   21 Jul 20 22:37 java-1.11.0-openjdk-amd64 -> java-11-openjdk-amd64
# lrwxrwxrwx 1 root root   21 Jul 22 21:02 java-1.17.0-openjdk-amd64 -> java-17-openjdk-amd64
# drwxr-xr-x 9 root root 4096 Aug  8 23:34 java-11-openjdk-amd64
# drwxr-xr-x 9 root root 4096 Aug  8 23:34 java-17-openjdk-amd64
# drwxr-xr-x 2 root root 4096 Aug  8 23:34 openjdk-11
# drwxr-xr-x 2 root root 4096 Aug  8 23:34 openjdk-17
```
But still, maybe it's better to use `/usr/lib/jvm/default-java` anyway to match the distro default R and because it's common for users to install `default-jdk` (not sure, might be wrong about this)? At least for RSPM, `default-jdk` is what we've been using for instructions on installing Java.